backend/danswer/configs/app_configs.py

```diff
@@ -201,7 +201,8 @@
 
 # Avoid to get archived pages
 CONFLUENCE_CONNECTOR_INDEX_ONLY_ACTIVE_PAGES = (
-    os.environ.get("CONFLUENCE_CONNECTOR_INDEX_ONLY_ACTIVE_PAGES", "").lower() == "true"
+    os.environ.get("CONFLUENCE_CONNECTOR_INDEX_ONLY_ACTIVE_PAGES", "true").lower()
```

Comment:
We probably should just not index archived pages. That feels like the more reasonable default.

Reply:
slightly weird approach no? If it's "" then it's False but if it's nothing, then it's true?